src/packages/excalidraw/README_NEXT.md

```diff
@@ -728,21 +730,25 @@ import { restoreElements } from "@excalidraw/excalidraw-next";
 
 This function will make sure all properties of element is correctly set and if any attribute is missing, it will be set to default value.
 
+When `localElements` are supplied, they are used to ensure that existing restored elements reuse `version` (and increment it), and regenerate `versionNonce`. Use this when you import the scene with elements which may include the ones already on the scene, to ensure that your reconciliation algorithm can detect them as newer. Required: supply `null`/`undefined` if not applicable.
+
```

Comment:
```suggestion
When `localElements` are supplied, they are used to ensure that existing restored elements reuse `version` (and increment it), and regenerate `versionNonce`. This will ensure that the updates to existing elements are treated as a new change when importing a scene hence ensuring that changes are detected correctly, for example in the Excalidraw app the changes are detected correctly if a scene with existing elements is imported during collaboration and thus reflected correctly for peers.
```

Tried simplifying (as the term `reconcilation algorithm` might not be clear  to users)  but not sure how much I could...

Reply:
It's meant for those who understand. Those who don't will ignore it, or research it if they want.

> for example in the Excalidraw app the changes are detected correctly if a scene with existing elements is imported during collaboration and thus reflected correctly for peers.

Dunno if this clears it up. I'd either keep the old version, or drop the note about reconciliation altogether if you think it's not helpful.

Reply:
Yes I would drop the reconciliation note for now as it will confuse the users. I am also not sure right now what would be the best way to explain here in simple so lets improve this later.